### PR TITLE
add SlideshowCaptions experiement

### PR DESCRIPTION
**Pull request by jamesgorrie**
## What does this change?
Adds the SlideshowCaptions experiment. 

Part of #24782

### CODE REVIEW

**Review by mxdvl — APPROVED** [review]
🗒️

File: common/app/experiments/Experiments.scala [hunk 1]
```diff
@@ -30,3 +30,12 @@ object StickyVideos
       sellByDate = LocalDate.of(2022, 6, 2),
       participationGroup = Perc0C,
     )
+
+object SlideshowCaptions
```
Comment (AshCorr) [inline common/app/experiments/Experiments.scala:34]: Does this need to be added to the above `allExperiments` list?

**Review by AshCorr — APPROVED** [review]
Looks good!

File: common/app/experiments/Experiments.scala (hunk 1 above)
Comment (jamesgorrie) [reply common/app/experiments/Experiments.scala:34]: It does we can do that in your PR.

**Pull request merged**